src/insert/spacefinder/spacefinder.ts

```diff
@@ -28,26 +34,49 @@ type SpacefinderItem = {
 type SpacefinderRules = {
 	bodySelector: string;
 	body?: HTMLElement | Document;
-	// selector(s) for the elements that we want to allow inserting ads above
+	/**
+	 * Selector(s) for the elements that we want to allow inserting ads above
+	 */
 	candidateSelector: string | string[];
```

Comment:
This is so that the VS Code feature of hovering over a property to see its description works.